docs/UserGuide.md

```diff
@@ -66,70 +74,95 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
+* Items with `…` after means that there can be **more than 1** item. It is necessary to specify the prefix
+ before every item.<br>
+  e.g. `[t/TAG]…` can be used as `t/healthy` or `t/healthy t/low calories`.
+
+* Items in the more format `[<character> MORE ITEM]` means there can be **more than 1** item, separated by the
+ specified character. It is not necessary to specify the prefix before each item. <br> 
+ e.g `instr/INSTRUCTION[. MORE INSTRUCTION]` can be used as `instr/Open baked beans and serve` or `instr/Boil pasta
+ . Fry pasta with meat sauce. Serve meat pasta.`.
+ e.g `n/[ MORE NAMES]` can be used as `n/salad` or `n/salad pie`.
 
 * `INDEX` **must be a positive integer** e.g. 1, 2, 3...<br>
-  e.g. the `INDEX` in `deleteR INDEX` and `editF INDEX` must be a positive integer that is present in the corresponding lists e.g. `deleteR 1` `editF 2`
+  e.g. the `INDEX` in `deleteR INDEX` and `editF INDEX` must be a positive integer that is present in the
+   corresponding lists e.g. `deleteR 1` `editF 2`.
 
-* Parameters can be in any order.<br>
+* Parameters can be in **any order**.<br>
   e.g. if the command specifies `n/NAME i/INGREDIENTS`, `i/INGREDIENTS n/NAME` is also acceptable.
 </div>
 <br><br>
 
 
 ## Recipe-related Commands
 
-The Recipe-related commands include `addR`, `recipes`, `deleteR`, `editR`, `selectR`, `close`, `searchR`, `recommend` and `clearR`. These are the commands in Wishful Shrinking that are relevant only to the Recipes.
+The Recipe-related commands include [`addR`](#add-recipe), [`recipes`](#list-recipe), [`deleteR`](#delete-recipe
+), [`editR`](#edit-recipe), [`selectR`](#select-recipe), [`close`](#close-recipe), [`searchR`](#search-recipe), 
+[`recommend`](#recommend-recipe) and [`clearR`](#clear-recipe). These are the commands in Wishful Shrinking that
+are relevant only to Recipes.
 <br><br><br>
 
-### Adding a recipe: `addR`
+### Adding a recipe: `addR` <a id="add-recipe"></a>
 
-Adds a recipe to Recipe list.
+Adds a recipe to Recipe List.
 
 <br>  The image below is what Wishful Shrinking looks like after you have added a recipe. <br><br>
 <img src="images/AddRecipeImage.png" width="550" height="300">
 <br><br><br>
 
-Format: `addR n/NAME i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]] c/CALORIES [img/IMAGE] inst/INSTRUCTION... [t/TAG]...`
+Format: `addR n/NAME i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]] c/CALORIES [img/IMAGE] inst
+/INSTRUCTION[. MORE INSTRUCTIONS] [t/TAG]...`
 
 * `INGREDIENT` can take in an optional `Quantity` e.g. i/Tomato -2 whole.
-    * Note: there is a mandatory space before `-`.
-    * Note: `QUANTITY` only accepts alphanumeric characters, forward slashes and full stops.
-* `CALORIES` **must be a positive integer** e.g. 100, 200, 300...
+<div markdown="span" class="alert alert-primary">:memo: **Note:**
+`QUANTITY` is separated with a mandatory space before `-` and after the dash, only accepts alphanumeric characters
```

Comment:
if notes really cant handle the markups, should remove the ` ? (but not sure if it will appear on the website)

Reply:
oh ya i'll change

Reply:
wait no i changed back because i think notes can handle the markups bc command format also has markups inside the div